frontend/passkeyNameModal.js

```diff
@@ -0,0 +1,257 @@
+import DOMPurify from 'dompurify';
+import M from 'materialize-css';
+import { displayToast, messageType } from './utils/toastMessages';
+
```

Comment:
Perhaps file should have header so that casual browser can see what it's for.

Reply:
Will do this in the next PR too! This will be refactored into a `modalUtils` file.

Reply:
The next PR is https://github.com/guardian/janus-app/pull/635